### PR TITLE
Rename "Stem Export" to "Audio Export" to match industry naming conventions.

### PR DESCRIPTION
**Pull request by stellar-aria**
A stem is a mixdown of a group of tracks, we currently only export clips and tracks, so it should really be called "Audio Export".

Changes:
- Documentation
- "STEMS" folder has been renamed to "EXPORTS"
- "ARRANGER" and "SONG" folders have been renamed to "TRACKS" and "CLIPS" respectively, to reflect what they contain rather than where they came from
- "ARRANGEMENT" filename has been rename to "MIXDOWN" to reflect what the file actually is (a stereo mixdown of the unmuted tracks being rendered)

### CODE REVIEW

File: docs/community_features.md [hunk 1]
```diff
@@ -1676,7 +1676,7 @@ different firmware
 
 [DX7 Synth Documentation]: https://github.com/SynthstromAudible/DelugeFirmware/blob/community/docs/features/dx_synth.md
 
-[Stem Export Documentation]: https://github.com/SynthstromAudible/DelugeFirmware/blob/community/docs/features/stem_export.md
+[Audio Export Documentation]: https://github.com/SynthstromAudible/DelugeFirmware/blob/community/docs/features/stem_export.md
```
Comment (seangoodvibes) [inline docs/community_features.md:1679]:
```suggestion
[Audio Export Documentation]: https://github.com/SynthstromAudible/DelugeFirmware/blob/community/docs/features/audio_export.md
```

Comment (stellar-aria) [reply docs/community_features.md]: fixed

File: docs/features/audio_export.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+> SYNTH_TRACK_PRESETNAME_000.WAV
+> ARRANGEMENT.WAV
+
+## Shortcuts to Start/Stop Exporting
+
+### Start Export
+
+- Hold `SAVE` + Press `RECORD` while Playback and Record are disabled to launch the export process
+- When the export is finished, a dialog will appear on the display that tells you that the export process has finished. Press `SELECT`, `BACK` or any `PAD` on the grid to exit the dialog.
+- Note 1: Tracks are exported without Master (Song) FX applied
```
Comment (seangoodvibes) [inline docs/features/audio_export.md:39]:
```suggestion
- Note 1: Master (Song) FX are excluded from the export by default (but can be included - see below)
```

File: docs/features/audio_export.md [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+- By default, files are rendered until silence is reached (mutable noise floor, ~70dB from peak) to allow for sound tails (e.g. delay, reverb) to be captured
+  - This can be turned off in the export configuration menu located at: `SONG\EXPORT AUDIO\CONFIGURE EXPORT\EXPORT TO SILENCE`
+- If silence is not reached within 60 seconds of playback stopping, then the recording will stop automatically as a safety precaution.
+
+### Normalization
+- By default, normalization is off. Normalization sets the peak of the renders to be at 0dB (as loud as possible without distorting).
+  - Normalization can be turned on in the export configuration menu located at: `SONG\EXPORT AUDIO\CONFIGURE EXPORT\NORMALIZATION`
+
+### Song FX
+- By default, song FX are excluded. They can be included in the export configuration menu located at: `SONG\EXPORT AUDIO\CONFIGURE EXPORT\SONG FX`
```
Comment (seangoodvibes) [inline docs/features/audio_export.md:70]:
```suggestion
- By default, song FX are excluded (these are the master FX in Song and Arranger Views when Affect Entire is enabled). They can be included in the export configuration menu located at: `SONG\EXPORT AUDIO\CONFIGURE EXPORT\SONG FX`
```

**Review by seangoodvibes — APPROVED** [review]

**Pull request merged**